Doc/using/windows.rst

```diff
@@ -602,6 +602,50 @@ existed)::
 
     C:\WINDOWS\system32;C:\WINDOWS;C:\Program Files\Python 3.9
 
+.. _win-utf8-mode:
+
+UTF-8 mode
+==========
+
+.. versionadded:: 3.7
+
+Windows still uses legacy encodings for the system encoding (the ANSI Code
+Page).  Python uses it for the default encoding of text files (e.g.
+:func:`locale.getpreferredencoding`).
+
+It may cause trouble because the UTF-8 is widely used on the internet
+and most Unix systems, including WSL (Windows Subsystem for Linux).
+
+You can use the UTF-8 mode to change the default text encoding to UTF-8.
+You can enable the UTF-8 mode via ``-X utf8`` command option, or
+``PYTHONUTF8=1`` environment variable.  See :envvar:`PYTHONUTF8` for
+the UTF-8 mode, and :ref:`setting-envvars` for how to modify
+environment variables.
+
+When UTF-8 mode is enabled:
+
+* :func:`locale.getpreferredencoding` returns ``'UTF-8'`` instead of
+  the legacy system encoding.  This function is used for the default
+  text encoding in many places, including :func:`open`, :class:`Popen`,
+  :meth:`Path.read_text`, etc.
+* :data:`sys.stdin`, :data:`sys.stdout`, and :data:`sys.stderr`
+  all use UTF-8 as their text encoding.
+* You can still use the legacy encoding via the "mbcs" codec.
```

Comment:
Perhaps to be clear, call this the "system codepage" instead of "legacy encoding". If the system codepage is set to UTF-8, then the MBS API is not using a legacy encoding.